exampleSite/static/admin/config.yml

```diff
@@ -30,6 +30,16 @@ collections:
           - {label: "Name", name: "name", widget: "string"}
           - {label: "Link", name: "job_link", widget: "string"}
           - {label: "Icon", name: "icon", widget: "string"}
+  # job landing page
```

Comment:
Assuming this is linked to #87:

1. The "Role Landing Page" bullets refer to `/content/jobs/${role}/_index.md` – for now, we only need to have a selection for the role (hard-coded since this is going to end up being a file collection just like other job example(s))
2. If you *wanted* to add body text, I wouldn't stop it.

If this is referring specifically to `/content/jobs/${role}/${job}` bullets, then we need to hardcode the title (or the `job_name` value that Scrubby is going with.

Reply:
Hey @nonowazu 

1. I have the changes in the other PR here: https://github.com/The-Balance-FFXIV/glam/pull/92. I believe it addresses your comments
2. I'll take a look at this again after reviewing Scrubby's changes.

Reply:
@nonowazu i've updated this PR.